features/steps/fedramp_extensions_steps.ts

```diff
@@ -665,4 +665,56 @@ Then("I should have both FAIL and PASS tests for constraint ID {string}", functi
     constraintId,
     `Constraint ${constraintId} is not in the extracted constraints list`
   );
+});
+
+Then('I should verify that all constraints follow the style guide constraint', async function () {
+  const baseDir = join(__dirname, '..', '..');
+  const constraintDir = join(baseDir, 'src', 'validations', 'constraints');
+  const styleGuidePath = join(baseDir, 'src', 'validations', 'styleguides', 'fedramp-constraint-style.xml');
+
+  const constraint_files = readdirSync(constraintDir).filter((file) => file.startsWith('fedramp') && file.endsWith('xml') );
+  const errors = [];
+
+  function filterOutBrackets(input) {
+    return input.replace(/\[.*?\]/g, '');
+  }
+
+  for (const file_name of constraint_files) {
+    const filePath = join(constraintDir, file_name.trim());
+    console.log(filePath);
+    try {
+      console.log(filePath);
+      const [result, error] = await executeOscalCliCommand('metaschema', [
+        'validate',
+        filePath,
+        '-c',
+        styleGuidePath,
+        '--disable-schema-validation'
+      ]);
```

Comment:
Now that we have rolled back to 2.2.0, can we please add something like this before final approval? Thanks!

```suggestion
  for (const file_name of constraint_files) {
    const filePath = join(constraintDir, file_name.trim());
    const styleGuideReportPath = join(styleGuideReportsDir, `${parse(filePath).name}.sarif`);
    console.log(filePath);
    try {
      console.log(filePath);
      const [result, error] = await executeOscalCliCommand('metaschema', [
        'validate',
        filePath,
        '-c',
        styleGuidePath,
        '-o',
        styleGuideReportPath,
        '--disable-schema-validation',
      ]);
```